packages/api/routes/routes.js

```diff
@@ -30,6 +37,17 @@ function register(app) {
   app.route("/list").get(cors, list());
 
   app.post("/autosync/forceSyncAll", cors, forceSyncAll());
-}
 
+  // API Keys
+  app
+    .route("/apikey")
+    .get(cors, getAPIKeysByUser())
+    .post(cors, createAPIKey())
+    .delete(cors, deleteAPIKey());
+
+  app
+    .route("/user")
```

Comment:
Where did the need for a `/user` path come from?

I'm looking at the directory structure and it looks like this:

 - apikeys
   - key
   - user

But the URL paths don't reflect that.  The directory structure is supposed to map directly onto the URL structure so it's easy to know which middleware is being used at a given path (yes, get/post/delete make it slightly less clear but it's still good practice).

Reply:
This was a late change that I made since it made more sense to collect everything related to apikeys under a single directory. I'm wondering if endpoints like `/apikeys/key` and `/apikeys/user` would make more sense.

Reply:
Cool, no worries.  I think it's fine to leave it like this and we can rearrange things post MVP.